flood_adapt/object_model/hazard/hazard.py

```diff
@@ -371,18 +364,19 @@ def preprocess_sfincs(
             )
             model.add_wl_bc(self.wl_ts)
 
-            # Generate and change discharge boundary condition
-            logging.info(
-                "Adding discharge boundary conditions if applicable to the overland flood model..."
-            )
-            self.add_discharge()
-            # convert to metric units
-            gui_units = UnitfulDischarge(
-                value=1.0, units=self.site.attrs.gui.default_discharge_units
-            )
-            conversion_factor = gui_units.convert(UnitTypesDischarge("m3/s"))
-            self.dis_ts = conversion_factor * self.dis_ts
-            model.add_dis_bc(self.dis_ts)
+            # ASSUMPTION: Order of the rivers is the same as the site.toml file
```

Comment:
Cant you check this? Check if the lists are the same

Reply:
I included a check in the SFINCS adapter that will throw a value error if the river coordinates are not the same or not in the same order. 
I also included that in the test test_hazard_preprocess_synthetic_discharge